geode-core/src/main/java/org/apache/geode/internal/CoreSanctionedSerializablesService.java

```diff
@@ -12,20 +12,16 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package org.apache.geode.codeAnalysis;
+package org.apache.geode.internal;
 
+import java.net.URL;
 
-import org.apache.geode.internal.memcached.GeodeMemcachedService;
+import org.apache.geode.internal.serialization.SanctionedSerializablesService;
 
-public class AnalyzeMemcachedSerializablesJUnitTest extends AnalyzeSerializablesJUnitTestBase {
+public class CoreSanctionedSerializablesService implements SanctionedSerializablesService {
```

Comment:
Local git diffs are correct, but github is incorrectly showing that AnalyzeMemcachedSerializablesJUnitTest changed to CoreSanctionedSerializablesService.